index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Setup and Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+  <div class="container">
+    <nav>
+      <a href="#" class="logo">My Logo</a>
```

Comment:
- [ ] Please kindly bring the logo out of the `nav` tag, a logo is not supposed to be part of the navigation links. 👍